databases/pgbouncer/Portfile

```
@@ -2,47 +2,30 @@
 
 PortSystem 1.0
 
-name			pgbouncer
-version			1.4.2
-revision        2
-categories		databases
-platforms		darwin
-maintainers		nomaintainer
-description		Lightweight connection pooler for PostgreSQL
-long_description	pgbouncer is a PostgreSQL connection pooler. \
-			Any target application can be connected to \
-			pgbouncer as if it were a PostgreSQL server, \
-			and pgbouncer will manage to connect to the \
-			server, or to reuse one of its existing	connections.
-
-homepage		http://pgbouncer.projects.postgresql.org/
-master_sites		http://pgfoundry.org/frs/download.php/3085/
-
-
-checksums           md5     5083110b5b4f2127234bfc7b1f451f8d \
-                    sha1    610198c3f2186d70267275f554e23bd43598e4b5 \
-                    rmd160  055796e23dd7029d3f4a2c882cf24399a5481cb1
-
-configure.env		PATH=$env(PATH):${prefix}/lib/postgresql90/bin
-
-depends_build		port:postgresql90
-depends_lib             port:libevent
-
-livecheck.type	regex
-livecheck.url	http://pgfoundry.org/frs/?group_id=1000258
-livecheck.regex	pgbouncer-(\[0-9\\.\]+)\\.tgz
-
-variant postgresql82 description {uses postgresql82 installation} {
-	depends_build		port:postgresql82
-	configure.env		PATH=$env(PATH):${prefix}/lib/postgresql82/bin
-}
-
-variant postgresql83 description {uses postgresql83 installation} {
-	depends_build		port:postgresql83
-	configure.env		PATH=$env(PATH):${prefix}/lib/postgresql83/bin
-}
-
-variant postgresql84 description {uses postgresql84 installation} {
-	depends_build		port:postgresql84
-	configure.env		PATH=$env(PATH):${prefix}/lib/postgresql84/bin
-}
+name                pgbouncer
+version             1.14.0
+categories          databases
+platforms           darwin
+license             ISC
+maintainers         nomaintainer
+description         Lightweight connection pooler for PostgreSQL
+long_description    pgbouncer is a PostgreSQL connection pooler. \
+            Any target application can be connected to \
+            pgbouncer as if it were a PostgreSQL server, \
+            and pgbouncer will manage to connect to the \
+            server, or to reuse one of its existing    connections.
```

It think this is more likely to be the presentation that was actually intended.
```suggestion
                    Any target application can be connected to \
                    pgbouncer as if it were a PostgreSQL server, \
                    and pgbouncer will manage to connect to the \
                    server, or to reuse one of its existing connections.
```